projects/core/src/product/facade/product-search.service.spec.ts

```diff
@@ -73,13 +73,13 @@ describe('ProductSearchService', () => {
   ));
 
   it('should be able to get search results', () => {
-    service.searchResults$.subscribe(results => {
+    service.getSearchResults().subscribe(results => {
```

Comment:
It should be like this:
```typescript
let result: type?;
 service.getSearchResults().subscribe(results => result = results).unsubscribe()
      expect(results).toEqual(mockSearchResults);
      expect(results).toEqual(mockSearchResults);
```

The point is that this test can silently pass if the nothing is returned from the subscribed observable.